apps/src/blockly/customBlocks/cdoBlockly/spritelabBlocks.js

```diff
@@ -241,19 +242,21 @@ export const blocks = {
           block.setUserVisible(false);
         },
         overrides: {
-          getVars(category) {
+          getVars() {
             return {};
           },
-          callType_: 'gamelab_behavior_get',
+          callType_: BLOCK_TYPES.behaviorGet,
         },
       });
 
     generator.behavior_definition = generator.procedures_defnoreturn;
 
-    Blockly.Procedures.DEFINITION_BLOCK_TYPES.push('behavior_definition');
+    Blockly.Procedures.DEFINITION_BLOCK_TYPES.push(
+      BLOCK_TYPES.behaviorDefinition
+    );
     Blockly.Variables.registerGetter(
       Blockly.BlockValueType.BEHAVIOR,
-      'gamelab_behavior_get'
+      BLOCK_TYPES.behaviorGet
```

Comment:
The changes here are probably fine, but I think we'd also want to delete this file sometime after the migration is completed.